tests/core-addon/integration/core_addon.js

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const firefox = require("selenium-webdriver/firefox");
+const { Builder, By, until } = require("selenium-webdriver");
+const assert = require("assert");
+
+let driver = null;
+const firefoxOptions = new firefox.Options();
+firefoxOptions.setPreference("xpinstall.signatures.required", false);
+// Unset this to run the UI (useful for local testing).
+firefoxOptions.headless();
+
+// This is the path to Firefox Nightly on Ubuntu with the Mozilla PPA
+if (process.platform === "linux") {
+  firefoxOptions.setBinary("/usr/bin/firefox-trunk");
+}
+
+describe("Selenium", function () {
+  // eslint-disable-next-line mocha/no-hooks-for-single-case
+  beforeEach(async function () {
+    driver = await new Builder()

```

Comment:
```suggestion
    this.driver = await new Builder()
```
... and we could do this